src/main.rs

```diff
@@ -468,6 +468,21 @@ fn run() -> Result<(), Error> {
         }
     }
 
+    if config.should_run(Step::Sdkman) {
```

Comment:
As I understand, SDKMAN! doesn't work on Windows at all, so this whole if should be inside `#[cfg(unix)]`

Reply:
Yes, initially I thought so but I think that there are ways to make it run under cygwin and that's why I removed the conditional compilation flag.

Reply:
I think that if Topgrade is compiled in Cygwin then Rust will set the `unix` configuration instead of `windows`. That's the assumption that I wrote the other steps with.

Reply:
But what if topgrade is compiled on the windows toolchain and then invoked from inside cygwin? Should it not be expected to work?

Reply:
I don't think so. If you install a native Python and launch it from cygwin, you won't be able to do `import signal` from that Python, because it's still a native Python. If someone wishes to use Topgrade inside Cygwin then they should compile it with the Rust version that comes with Cygwin, and in that case I believe the toolchain will be considered as unix.

Reply:
Added the conditional compilation flag to the run_sdkman call.